guardian/models.py

```diff
@@ -64,16 +64,16 @@ class UserObjectPermissionBase(BaseObjectPermission):
     **Manager**: :manager:`UserObjectPermissionManager`
     """
     user = models.ForeignKey(user_model_label, on_delete=models.CASCADE)
+    origin = models.ForeignKey(Origin, on_delete=models.CASCADE, null=True)
```

Comment:
Moving this here so that sub-classes pick this up as it is now required. Note the migration doesn't result in a schema change, just django bookeeping.